MariaDB/server
# MDEV-32546: Sort Debian deb .install filenames

- [x] *The Jira issue number for this PR is: MDEV-32546*

## Description
Fix Debian .install files to alphabetically ordered from mostly ordered state.

## How can this PR be tested?
Check with `sort` that `.install` file is reordered and nothing is missing is translation like:
Example:
```
sort -u mariadb-server.install > mariadb-server.install.1; diff -urN mariadb-server.install mariadb-server.install.1
```

Easiest way to check changes would probably be using `git difftool`

## Basing the PR against the correct MariaDB version

## PR quality check
